packages/nvidia_nat_openpipe_art/src/nat/plugins/openpipe/trainer.py

```
@@ -0,0 +1,659 @@
+                else:
+                    # No trajectories collected, create a dummy status
+                    job_statuses.append(
+                        TrainingJobStatus(run_id=self._run_id,
+                                          backend="openpipe-art",
+                                          status=TrainingStatusEnum.COMPLETED,
+                                          message="No trajectories to train on",
+                                          metadata={"epoch": epoch}))
+
+                logger.info(f"Completed epoch {epoch + 1}/{num_epochs}")
+
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)